### DISCUSSION TITLE
Verifying github.io domains for orginizations

### DISCUSSION THREAD

**Discussion by Quantalabs** [community/community#23381] [Other Feature Feedback, Questions, & Ideas]
<p>I want to verify my org domain, which is default pages domain, <code>user.github.io</code>, but how do I verify the name on GitHub? Will github support creating a txt file on the repo? And is it okay to make this public?</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="Quantalabs" data-post="1" data-topic="190294">
<div class="title">
<div class="quote-controls"></div>
 Quantalabs:</div>
<blockquote>
<p>how do I verify the name on GitHub?</p>
</blockquote>
</aside>
<p>A quick and dirty way to do this is to click on the repository :gear: icon near the <strong>About</strong> column next to the files bar, which exposes a settings pop-up containing the <strong>Website</strong> field — when this field is empty, you’ll see a greyed out suggestion with the default GHPages link for the repository.</p>

**Reply by nzewunwagrace94** [reply]
Thank you for joining. Your coupon has already been applied and is active. If you believe this is an error please contact your administrator. I got this message when I wanted to apply for a coupon, meanwhile, I have requested it previously. how do I rectify t
his to get my student coupon to purchase my domain name

**Reply by nzewunwagrace94** [reply]
i am trying to reach out to the adminitrator, but can't find where to do that

**Comment by Quantalabs** [comment]
<p>I’m not sure I understand. Could you please provide a screenshot?</p>

**Comment by Quantalabs** [comment]
<p>Oh, I understand, however, I need to verify the org domain, not add a domain to the repository. For example with bootstrap:<br>
<img src="https://user-images.githubusercontent.com/55121845/181076672-0b10c7cf-76f0-435e-9d34-734d688fbd9f.png" alt="image" data-base62-sha1="k3h17nOCIH5RF2MxBpJJCnuNrqo" width="662" height="209"></p>

**Comment by tajmone** [comment, marked as answer]
<p>Sorry, I though by org you meant the profile website for an organization on GitHub.</p>
<p>If you click on the <kbd>Verified</kbd> label you find a link for more info on how it works:</p>
<p><a href="https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization</a></p>
